### ISSUE TITLE
Optional checking & notifying user about updated modules

### ISSUE THREAD

**Issue by tuomas2** [AndBible/and-bible#2351]
Perhaps weekly check which can be disabled in settings.
User is asked opinion first time downloading something.

**Comment by agrogers** [comment]
And also notifying about upcoming releases etc

**Comment by tuomas2** [comment]
Good idea.

We could have a file in our data repository for custom notifications, that is read similarly once a week (if feature is enabled). 
Should add a translatable string about this.
Something like this:

> A new release is about to be released starting from %s. See our [trailer video] to see new features. 
> 
> After release, this version is left unmaintained, i.e. it will not receive any bug fixes any more. 
> If don't want to upgrade, please disable automatic upgrades from your app store (Google Play / F-Droid etc). 
>

**Comment by tuomas2** [comment]
Moving this idea to another ticket...

**Comment by meichthys** [comment]
Related to #153 (Maybe Duplicate?)

**Comment by tuomas2** [comment]
Yes, duplicate. Closing this and adding another to "Wishing well" project.
